doc/manual/src/contributing/testing.md

```diff
@@ -10,6 +10,13 @@ under `src/{library_name}/tests` using the
 You can run the whole testsuite with `make check`, or the tests for a specific component with `make libfoo-tests_RUN`.
 Finer-grained filtering is also possible using the [--gtest_filter](https://google.github.io/googletest/advanced.html#running-a-subset-of-the-tests) command-line option, or the `GTEST_FILTER` environment variable.
 
+### Characterization testing
+
+See below for a broder discussion.
+`_NIX_TEST_ACCEPT=1` is also used.
+The data is in `unit-test-data`;
+the path to that directory is passed to the unit test executable with the environment variable `UNIT_TEST_ENV`.
+
```

Comment:
I don't really have a clue as to what I should take out of this paragraph. What I'd expect to know is

- How do I run these tests?
- How do I edit one or add a new one?
- How do I regenerate the golden files?

Reply:
1 and 3 are now covered. (They are just unit tests, they are run like any other units tests). 2 I am not sure it is good to include yet because there is no *general* pattern yet.

I would like to convert a bunch of existing unit tests to use this sort of thing down the road, and at that point I think we'll have enough generalized infra to answer this question properly.